tests/mongo-tools/dump_restore_test.go

```diff
@@ -26,8 +26,8 @@ func TestDumpRestore(t *testing.T) {
 	t.Parallel()
 
 	containerSourceRoot := "/dumps/mongodb-sample-databases/dump"
-	localTestsRoot := filepath.Join("..", "..", "dumps", "tests")
-	containerTestsRoot := "/dumps/tests"
+	localTestsRoot := filepath.Join("..", "..", "dumps", "mongoexport_tests")
+	containerTestsRoot := "/dumps/mongoexport_tests"
```

Comment:
I think `mongoexport_tests` and `mongodump_tests` are swapped between there and export_import_test.go

Reply:
whoops... nice catch!